src/pull-request/pull-request-chunk.html

```diff
@@ -54,18 +58,57 @@
       :host(:hover) header iron-icon {
         opacity: 1;
       }
+      paper-input.title {
+        --paper-input-container: {
+          padding: 0 0 1em 0;
+        };
+        --paper-input-container-input: {
+          font-size: 1.5em;
+        };
+        --paper-input-container-label: {
+          font-size: 1.5em;
+          color: var(--secondary-text-color);
+        };
+        --paper-input-container-label-floating: {
+          font-size: 1em;
+        }
+        --paper-input-container-color: var(--secondary-background-color);
+        --paper-input-container-focus-color: var(--anchor-color);
+      }
     </style>
     <header>
       <iron-icon icon="icons:remove-circle-outline" on-tap="_toggleCollapse"></iron-icon>
-      <h2>[[title]]</h2>
-      <marked-github-element markdown="[[description]]" owner="[[owner]]" name="[[name]]"></marked-github-element>
+      <paper-input hidden$="[[!editable]]" class="title" label="Enter a title" value="{{title}}"></paper-input>
+      <h2 hidden$="[[editable]]">[[title]]</h2>
+      <markdown-input
+        disabled="[[!editable]]"
+        label="Enter a description for these changes"
+        markdown="{{description}}"
+        owner="[[owner]]"
+        name="[[name]]"
+        ></markdown-input>
     </header>
-    <sortable-js group="chunk" draggable="pull-request-diff-highlight" on-remove="_diffRemoved">
-      <template is="dom-repeat" items="[[chunk.diff]]">
-        <pull-request-diff-highlight code="[[item.changes]]" file="[[item.file]]"></pull-request-diff-highlight>
-      </template>
-    </sortable-js>
-    <iron-icon icon="icons:arrow-downward" class="toggle-comments-icon" on-tap="_toggleComments"></iron-icon>
+    <template is="dom-if" if="[[editable]]">
```

Comment:
Instead of a `dom-if` with copy-paste code, we should use the `disabled` property of `sortable-js`. E.g. `disabled="[[!editable]]"`